sharepoint/sharepoint-ps/sharepoint-online/Add-SPOTenantRestrictedSearchAllowedList.md

```diff
@@ -0,0 +1,103 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/add-spotenantrestrictedsearchallowedlist
+applicable: SharePoint Online
+title: Add-SPOTenantRestrictedSearchAllowedList
+schema: 2.0.0
+author: praporwal
+ms.author: praporwal
+ms.reviewer:
+---
+
+# Add-SPOTenantRestrictedSearchAllowedList
+
+## SYNOPSIS
+
+Add a list of sites to the allowed list.
+
+## SYNTAX
+
+```powershell
+Add-SPOTenantRestrictedSearchAllowedList -SitesList <List[string]> [<CommonParameters>]
+```
+
+```powershell
+Add-SPOTenantRestrictedSearchAllowedList -SitesListFileUrl <string> [-ContainsHeader <bool>] [<CommonParameters>]
```

Comment:
>-ContainsHeader [](http://example.com/codeflow?start=69&length=15)

curious why we didn't make this a switch instead of a bool?

Reply:
Just got missed I guess.